shell/platform/darwin/ios/framework/Source/FlutterView.mm

```diff
@@ -96,7 +101,7 @@ - (void)layoutSubviews {
     layer.allowsGroupOpacity = YES;
     layer.contentsScale = screenScale;
     layer.rasterizationScale = screenScale;
-    layer.framebufferOnly = flutter::Settings::kSurfaceDataAccessible ? NO : YES;
```

Comment:
This value should match what we set in https://github.com/flutter/engine/blob/3303a3e040b1ea27a4a732612eae32c0eaf5d21f/shell/platform/darwin/ios/ios_surface_metal_impeller.mm#L57